packages/suite/src/components/suite/Notifications/UpdateBridge.tsx

```diff
@@ -12,6 +13,11 @@ interface Props {
 
 const UpdateBridge = ({ transport, goto }: Props) => {
     if (!transport || !transport.outdated) return null;
+    if (isDesktop()) {
+        // we can work with these versions of bridge too
+        if (transport.version && ['2.0.27', '2.0.28', '2.0.29'].includes(transport.version))
```

Comment:
So your intention is to cover cases when there is an even older version. In such case we want users to see the notification and update?